nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/CommonClassLoaderServiceImpl.java

```diff
@@ -172,12 +172,7 @@ public String getCommonClassPath() {
     }
 
     private static String urlsToClassPath(Stream<URL> urls) {
-        return urls
-                .map(URL::getFile)
-                .filter(Predicate.not(String::isBlank))
-                .map(File::new)
-                .map(File::getAbsolutePath)
-                .collect(Collectors.joining(File.pathSeparator));
+        return urls.map(FileUtils::toFile).map(File::getAbsolutePath).collect(Collectors.joining(File.pathSeparator));
```

Comment:
I'd keep original line breaks to show that only part of the method changed, not the whole one.

Reply:
It is under 120 characters border, so it is not so hard to read. I would agree otherwise.